rllib/tuned_examples/ppo/multi_agent_pendulum_ppo.py

```diff
@@ -1,7 +1,12 @@
 from ray.rllib.algorithms.ppo import PPOConfig
 from ray.rllib.examples.envs.classes.multi_agent import MultiAgentPendulum
+from ray.rllib.utils.test_utils import add_rllib_example_script_args
 from ray.tune.registry import register_env
 
+parser = add_rllib_example_script_args()
+# Use `parser` to add your own custom command line options to this script
+# and (if needed) use their values toset up `config` below.
+args = parser.parse_args()
 
 register_env("multi_agent_pendulum", lambda _: MultiAgentPendulum({"num_agents": 2}))
```

Comment:
@sven1977 I think we must use here the `"num_agents": args.num_agents`?

Reply:
fixed